packages/rich-text/src/component/use-copy-handler.js

```diff
@@ -41,11 +42,17 @@ export function useCopyHandler( props ) {
 				multilineTag || ''
 			);
 			event.preventDefault();
+
+			if ( event.type === 'cut' ) {
+				ownerDocument.execCommand( 'delete' );
```

Comment:
I just noticed that this interface has been deprecated. :/

https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand

Reply:
Yes, but there is no alternative and realistically browsers will not drop support because of compatibility. Many editors rely on this. It's similar to `keyCode` being deprecated.

Reply:
Ideally, we don't have to call `event.preventDefault()` here and let the browser remove the selected text, but for some reason I then can't set the clipboard data. I'll look into it more in a bit.

Reply:
Ok, we can check. I guess it's not the end of the world if it copies but does not delete the text.